### ISSUE TITLE
Is it config.yaml or settings.yml?

### ISSUE THREAD

**Issue by stuartmac** [bartongroup/slivka#138]
@warownia1 did you rename the main configuration file from config.yaml to settings.yml at some point? 

My Slivka instance was initialised with a settings.yml file, and I see this in the project_template in the repository, but the documentation refers to config.yaml. Here for example: https://bartongroup.github.io/slivka/specification.html.

Should the documentation be updated?

**Comment by warownia1** [comment]
Good spot, maybe I should keep it more consistent.
But, the very first section where the *config.yaml* file is introduced
writes:

> "It can alternatively be named *settings.yaml* and either *.yaml* or
> *.yml* extension is allowed."

**Comment by stuartmac** [comment]
I think consistency is key. Seems like an example where flexibility comes with an overhead that might not be worth it.

If possible I'd suggest having a single allowed name for this file, but if you need to maintain backwards compatibility, maybe adding an inline comment to the project structure tree in the docs can help? Like below:

```
<project-root>/
├── config.yaml  # or settings.yaml
├── manage.py
├── wsgi.py
├── scripts/
│  ├── example.py
│  └── selectors.py
├── services/
│  └── example.service.yaml
└── static/
   ├── openapi.yaml
   └── redoc-index.html
```

**Comment by warownia1** [comment]
Fixed by 6d69eeec4810c0aea29b061f5810e092a9d667a8
